requirements_parser/requirements_parser.py

```diff
@@ -0,0 +1,40 @@
+class RequirementsParser:
+
+    def open_requirements(self, path):
+        """
+        adding to dict all the dependencies
+
+        Parameters:
+        file (str): requirements.txt file location / requirements.txt file name
+
+        Returns:
+        out_dict(dict): contains all the dependencies
```

Comment:
again... in your docs you said that out_dict is dict but is set! in addition, what is out? you mean output?